strawberry_django/filters.py

```diff
@@ -0,0 +1,121 @@
+from typing import Type, Optional, List
+import strawberry
+from graphql import GraphQLError
+from strawberry.arguments import UNSET, is_unset
+
+
+__all__ = [
+    "FilterSetError",
+    "apply_filter",
+    "create_filter",
+    "get_filter_field_type",
+    "set_filter_field_type",
+]
+
+
+try:
+    import django_filters
+except ModuleNotFoundError as e:
+    raise ModuleNotFoundError(
+        'You need to install django-filters to use "strawberry_django.filters". '
+        "See https://django-filter.readthedocs.io/"
+    )
```

Comment:
This exception breaks whole `strawberry_django` package if `django_filters`is not installed. Would following work?
```python
try:
    import django_filters
except ModuleNotFoundError:
    django_filters = None

def check_django_filters_package():
    if django_filters is None:
        raise ModuleNotFoundError(
            'You need to install django-filters to use "strawberry_django.filters". '
            "See https://django-filter.readthedocs.io/"
        )

def create_filter():
    check_django_filters_package()
    ...
```

Reply:
Yeah good point

Reply:
Nice. I'm getting very useful error when trying to use strawberry_django.filter without having django-filters installed!

Do you have ideas how we could unit test that?

Reply:
I opened another discussion for unit testing. This has been resolved. Thanks!